docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -441,26 +458,16 @@ spec:
         readOnly: true
 "
 ----
-Note that you should retain the original `OpenStackDataPlaneNodeSet` services
-composition, except the inserted `ceph-client` service.
-
-+ Ensure that the `ovn-controller` settings that are configured in the `OpenStackDataPlaneNodeSet` CR are the same as were set in the Compute nodes before adoption. This configuration is stored in the `external_ids` column in the `Open_vSwitch` table in the Open vSwitch database:
-+
-----
-ovs-vsctl list Open .
-...
-external_ids        : {hostname=standalone.localdomain, ovn-bridge=br-int, ovn-bridge-mappings=<bridge_mappings>, ovn-chassis-mac-mappings="datacentre:1e:0a:bb:e6:7c:ad", ovn-encap-ip="172.19.0.100", ovn-encap-tos="0", ovn-encap-type=geneve, ovn-match-northd-version=False, ovn-monitor-all=True, ovn-ofctrl-wait-before-clear="8000", ovn-openflow-probe-interval="60", ovn-remote="tcp:ovsdbserver-sb.openstack.svc:6642", ovn-remote-probe-interval="60000", rundir="/var/run/openvswitch", system-id="2eec68e6-aa21-4c95-a868-31aeafc11736"}
-...
-----
-
 +
-* Replace `<bridge_mappings>` with the value of the bridge mappings in your configuration, for example, `"datacentre:br-ctlplane"`.
+[NOTE]
+Ensure that you retain the original `OpenStackDataPlaneNodeSet` services
```

Comment:
Does “original OpenStackDataPlaneNodeSet  services" refer to the one deployed in step 7?